load-test-framework/src/main/java/com/google/pubsub/flic/controllers/Client.java

```diff
@@ -81,6 +81,8 @@ static String getTopicSuffix(ClientType clientType) {
       case CPS_GCLOUD_PUBLISHER:
```

Comment:
Should this be CPS_GCLOUD_JAVA_PUBLISHER now? Same goes for CPS_GCLOUD_SUBSCRIBER. Actually, what is the 'GCLOUD' referring to? The client library? If so, does CPS_GCLOUD_PYTHON_PUBLISHER even make sense?